.github/workflows/links.yml

```
@@ -0,0 +1,22 @@
+name: Links
+
+on:
+  push:
+  pull_request:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "00 18 * * *"
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1.0.9
+        with:
+          args: --verbose --no-progress **/*.md **/*.html
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

```suggestion
      - name: Link Checker
        uses: lycheeverse/lychee-action@v1.0.9
        with:
          args: --verbose --no-progress **/*.md **/*.html
        env:
          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
      -if: failure()
      - if: success()
```
Let's try this, [I also opened a PR on the repo for the link-checker action](https://github.com/lycheeverse/lychee-action/issues/58).

---

Alternatively, the maintainer of lychee-action suggests:

```suggestion
      - name: Link Checker
        uses: lycheeverse/lychee-action@v1.0.9
        with:
          args: --verbose --no-progress **/*.md **/*.html
        env:
          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
     - name: Fail if there were link errors
        run: exit ${{ steps.lychee.outputs.exit_code }}
```

---

I think we should check this once. The use case [here](https://github.com/lycheeverse/lychee-action/issues/20) is similar to ours